lib/index.js

```diff
@@ -213,7 +213,7 @@ Client.prototype.parseEvent = function (eObj) {
     var id = eObj.actionid;
     if (id !== undefined && this.responses[id] !== undefined && this.callbacks[id] !== undefined) {
         this.emit('responseEvent', eObj);
-        if (!this.responses[id].events) {
+        if (!Array.isArray(this.responses[id].events)) {
```

Comment:
This should not really help, as I think. Cause exception is `TypeError: Cannot read property 'push' of undefined` that means, that `this.responses[id].events === undefined` that is falsy